src/transformers/models/reformer/modeling_reformer.py

```diff
@@ -2360,13 +2357,20 @@ def forward(
 
         Returns:
 
+        <Tip warning={true}>
```

Comment:
I think this could use `_CHECKPOINT_FOR_DOC` but remove the parts that check the expected output values, and we don't need this warning?

At least, this is what you have done for some models that used `tiny-random-xxx` checkpoints.

Reply:
(Hmm, it raises a question: when we use real checkpoints which don't contain head weights, so far we don't include the output checks, but we don't do any warning. The warning introduced in your last PR is only used for 2 models so far where no small enough real checkpoints exist).

Would like to hear from you regarding this - as I remember one major reason is to avoid user confusion regarding the results they get.

Reply:
Here the checkpoint fails because it has `is_decoder=True` and we need `False` to use the MLM head.

Reply:
OK, thanks for explaining :-)